modules/installation-mirror-repository.adoc

```diff
@@ -0,0 +1,87 @@
+can access the certificate and credentials that you configured.
+* You downloaded the pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page and modified it to include authentication to your mirror repository.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Review the
+link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Quay.io]
+repository to determine the version of {product-title} that you want to install.
+
+. Set the required environment variables:
++
+----
+$ export OCP_RELEASE=<release_version> <1>
+$ export LOCAL_REGISTRY='<bastion_host_name>:<local_registry_host_port>' <2>
+$ export LOCAL_REPOSITORY='<repository_name>' <3>
+$ export PRODUCT_REPO='openshift-release-dev' <4>
+$ export LOCAL_SECRET_JSON='<path_to_pull_secret>' <5>
+$ export RELEASE_NAME="ocp-release" <6>
+----
+<1> For `<release_version>`, specify the version number of {product-title} to
+install, such as `4.2.0`.
+<2> For `<bastion_host_name>`, specify the registry domain name for your mirror
+repository, and for `<local_registry_host_port>`, specify the port that it
+serves content on.
+<3> For `<repository_name>`, specify the name of the repository to create in your
+registry, such as `ocp4/openshift4`.
+<4> The repository to mirror. For a production release, you must specify
+`openshift-release-dev`.
+<5> For `<path_to_pull_secret>`, specify the absolute path to and file name of
+the pull secret for your mirror registry that you created.
+<6> The release mirror. For a production release, you must specify
+`ocp-release`.
+
+. Mirror the repository:
++
+----
+$ oc adm -a ${LOCAL_SECRET_JSON} release mirror \
+     --from=quay.io/${UPSTREAM_REPO}/${RELEASE_NAME}:${OCP_RELEASE} \
+     --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} \
+     --to-release-image=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}
+----
++
+This command pulls the release information as a digest. The command output
+resembles the following text:
++
+----
+Success
+Update image:  <bastion_host_name>:<local_registry_host_port>/<local_registry>/<local_repository>:<release_version>
+Mirror prefix: <bastion_host_name>:<local_registry_host_port>/<local_registry>/<local_repository>
+
+To use the new mirrored repository to install, add the following section to the install-config.yaml:
+
+imageContentSources:
+- mirrors:
+  - <bastion_host_name>:<local_registry_host_port>/<repository_name>/release
+  source: quay.io/openshift-release/ocp-release
+- mirrors:
+  - <bastion_host_name>:<local_registry_host_port>/<repository_name>/release
+  source: registry.svc.ci.openshift.org/<local_registry>/<local_repository>
+...
+----
+
+. Record the `imageContentSources` section from the output of the previous
+command. This information is required during {product-title} installation.
+
+. Create the release package from the content that you mirrored:
```

Comment:
"Create the release package" -> "Extract the installation program"

Reply:
And maybe "to pin it to the mirrored release" or some such?  We don't want folks thinking, "oh, I already have an `openshift-install`" and skipping this step.